docs/source/recipes/librispeech/conformer_ctc.rst

```diff
@@ -292,9 +292,18 @@ The commonly used options are:
 
   - ``--method``
 
-    This specifies the decoding method.
+    This specifies the decoding method. This script support seven decoding methods. 
+    As for ctc decoding, it uses a sentence piece model to convert word pieces to words. 
+    And it needs neither a lexicon nor an n-gram LM.
+    
+    For example, the following command uses CTC topology for rescoring:
+    
+    .. code-block::
+
+      $ cd egs/librispeech/ASR
+      $ ./conformer_ctc/decode.py --method ctc-decoding --max-duration 300 --bucketing-sampler False
```

Comment:
You don't need to specify the option `--bucketing-sampler`. It is used only in the training script.